var/spack/repos/builtin/packages/mvapich2-gdr/package.py

```diff
@@ -146,11 +147,6 @@ def process_manager_options(self):
             )
         return opts
 
-    def setup_build_environment(self, env):
-        # mvapich2 configure fails when F90 and F90FLAGS are set
```

Comment:
This seems like something that should persist here.

Reply:
https://github.com/spack/spack/pull/48276#discussion_r1898717944